src/app/theme-chooser/themes.scss

```diff
@@ -0,0 +1,380 @@
+  $palette: $light-theme-palette,
+  $header-background: $secondary-light-700,
+  $header-text-color:$primary-light-600,
+  $cell-selected-background: $secondary-light-600,
+  $row-hover-background: $secondary-light-100,
+  $header-border-color: $primary-light-600
+);
+
+// Define dark theme for the grid
+$dark-grid: igx-grid-theme(
+  $palette: $dark-theme-palette,
+  $content-background: $secondary-dark-100,
+  $header-background: $primary-dark-500,
+  $header-text-color: $secondary-dark-600,
+  $cell-selected-background: $secondary-dark-600,
+  $cell-selected-text-color: $primary-dark-600,
+  $row-hover-background: $primary-dark-100,
+  $header-border-color: $primary-dark-600
+);
+
+// Define black theme for the grid
+$black-grid: igx-grid-theme(
+  $palette: $black-theme-palette,
+  $content-background:  $primary-black-100,
+  $header-background: $secondary-black-700,
+  $cell-selected-background: $secondary-black-600,
+  $row-hover-background: $secondary-black-100,
+  $header-border-color: $primary-black-600
+);
+
+// Define light theme for the grid-paginator
+$light-grid-paginator: igx-grid-paginator-theme(
+  $palette: $light-theme-palette,
+  $background-color:$primary-light-100,
+  $border-color: $secondary-light-500
+);
+
+// Define dark theme for the grid-paginator
+$dark-grid-paginator: igx-grid-paginator-theme(
+  $palette: $dark-theme-palette,
+  $text-color: $secondary-dark-700,
+  $background-color:$primary-dark-500,
+  $border-color: $secondary-dark-500
+);
+
+// Define black theme for the grid-paginator
+$black-grid-paginator: igx-grid-paginator-theme(
+  $palette: $black-theme-palette,
+  $text-color: $secondary-black-700,
+  $background-color:$primary-black-500,
+  $border-color: $secondary-black-500
+);
+
+// Define light theme for the grid-filtering
+$light-grid-filtering: igx-grid-filtering-theme(
+  $palette: $light-theme-palette,
+  $toggle-icon-color: $primary-light-500,
+  $toggle-active-background:  $primary-light-100,
+  $toggle-hover-background:  $secondary-light-500,
+  $menu-background: $primary-light-100,
+  $menu-text-color: $secondary-light-500,
+  $menu-button-text-color: $secondary-light-500,
+  $menu-button-disabled-text-color: $secondary-light-500	
+);
+
+// Define dark theme for the grid-filtering
+$dark-grid-filtering: igx-grid-filtering-theme(
+  $palette: $dark-theme-palette,
+  $menu-background: $primary-dark-100,
+  $toggle-hover-background: $primary-dark-100,
+  $menu-text-color: $secondary-dark-500,
+  $menu-button-text-color: $secondary-dark-500,
+  $menu-button-disabled-text-color: $secondary-dark-500
+  );
+
+// Define black theme for the grid-filtering
+$black-grid-filtering: igx-grid-filtering-theme(
+  $palette: $black-theme-palette,
+  $toggle-background: $primary-black-100,
+  $menu-background: $primary-black-100,
+  $menu-text-color: $secondary-black-500,
+  $menu-button-text-color: $secondary-black-500,
+  $menu-button-disabled-text-color: $secondary-black-700
+  );
```

Comment:
Can you split this file into two files? One that defines the themes, then this file imports the defined themes, and uses them in the structural css?